doc/JSONs.md

```diff
@@ -42,28 +42,18 @@ The `processed_metadata.json` file contains an array of objects, each representi
 
 ### Extraction Method
 
-| Attribute            | Type   | Description                                           | Values    | Example                           |
-|----------------------|--------|-------------------------------------------------------|-----------|-----------------------------------|
-| **type**             | String | The type of extraction method                         | regex     | `"regex"`                         |
-| **location**         | String | The location in the document where the URL was found  |           | `"DESCRIPTION"`                   |
-| **location_type**    | String | The type of location in the document                  |           | `"DOI"`                           |
-| **source**           | String | The source used for extraction                        |           | `"SOMEF"`                         |
-| **source_paragraph** | String | The paragraph in the source where the URL was found   |           | `"The code is available online."` |
+| Attribute            | Type   | Description                                             | Included in             | Values                                                                      | Example                              |
+|----------------------|--------|---------------------------------------------------------|-------------------------|-----------------------------------------------------------------------------|--------------------------------------|
+| **type**             | String | The type of extraction method                           | regex, unidir and bidir | regex, unidir, bidir                                                        | `"regex"`                            |
+| **location**         | String | The location in the document where the URL was found    | unidir, bidir           | ZENODO, DESCRIPTION, RELATED_PAPERS for bidir and the paper path for unidir | `"DESCRIPTION"`                      |
+| **location_type**    | String | The type of location in the document                    | unidir, bidir           | DOI, ARXIV, CFF, BIBTEX, TEXT for bidir and PAPER for unidir                | `"DOI"`                              |
+| **source**           | String | The source used for extraction                          | bidir                   | PAPER, SSKG, SOMEF                                                          | `"SOMEF"`                            |
```

Comment:
please explain what SSKG is. I think it should be RSEF. Also what does that mean?